browser/src/react-redux/App/User/User.reducer.js

```diff
@@ -0,0 +1,10 @@
+const initialState = null
+
+export default (state = initialState, action) => {
+  switch (action.type) {
+    case 'NEW_USER_SUBMITTED':
```

Comment:
Now that our action `setUser` is shared, let's go ahead and make this more generic and match the action name. Let's change `NEW_USER_SUBMITTED` to `SET_USER`.